test/e2e/common_test.go

```diff
@@ -33,16 +32,17 @@ import (
 )
 
 var (
-	PODMAN_BINARY      string
-	CONMON_BINARY      string
-	CNI_CONFIG_DIR     string
-	RUNC_BINARY        string
-	INTEGRATION_ROOT   string
-	CGROUP_MANAGER     = "systemd"
-	ARTIFACT_DIR       = "/tmp/.artifacts"
-	RESTORE_IMAGES     = []string{ALPINE, BB, nginx}
+	//lint:ignore ST1003
+	PODMAN_BINARY      string                        //nolint
```

Comment:
I would prefer If you use the `//nolint:<name>` syntax to only disable the check with the given name, e.g. `//nolint:golint` in this case

Reply:
I tried that, too, and it just changed the error to something else. But I'll try again (next week) and be less lazy.